k2/csrc/fsa.h

```diff
@@ -219,11 +213,11 @@ class CfsaVec {
 
   int32_t NumFsas() const { return num_fsas_; }
 
-  Cfsa operator[] (int32_t f) const;
+  Cfsa operator[](int32_t f) const;
 
  private:
-  CfsaVec &operator = (const CfsaVec &);  // Disable
-  CfsaVec(const CfsaVec&);  // Disable
+  CfsaVec &operator=(const CfsaVec &);  // Disable
```

Comment:
The modern style with `=delete` is more readable.

```cpp
public:
  CfsaVec &operator=(const CfsaVec &) = delete;

  CfsaVec(const CfsaVec &) = delete;
```

`public` is to suppress the warnings from clang-tidy.


----

I've changed this in my pullrequest.

Reply:
OK, thanks (I'll reslove it here)!